Dockerfile

```diff
@@ -8,11 +8,11 @@ ARG XX_VERSION=1.5.0
 ARG VPNKIT_VERSION=0.5.0
 
 ARG DOCKERCLI_REPOSITORY="https://github.com/docker/cli.git"
-ARG DOCKERCLI_VERSION=v27.0.2
+ARG DOCKERCLI_VERSION=v27.2.1
 # cli version used for integration-cli tests
 ARG DOCKERCLI_INTEGRATION_REPOSITORY="https://github.com/docker/cli.git"
 ARG DOCKERCLI_INTEGRATION_VERSION=v17.06.2-ce
-ARG BUILDX_VERSION=0.16.1
+ARG BUILDX_VERSION=0.17.0
 ARG COMPOSE_VERSION=v2.29.0
```

Comment:
Compose can also be updated to v2.29.2; https://github.com/docker/compose/releases/tag/v2.29.2